cmd/kwok/main.go

```diff
@@ -28,6 +29,7 @@ func main() {
 	command := cmd.NewCommand(logger)
 	err := command.Execute()
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
```

Comment:
```
W1118 11:00:42.067335   81145 client_config.go:617] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
W1118 11:00:42.067360   81145 client_config.go:622] error creating inClusterConfig, falling back to default config: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined
Error: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable
error: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable
exit status 1
```

Reply:
Maybe this could be removed, or the built-in default error output could be disabled?

Reply:
we can remove it, because cobra will print error info when runE returns an error